provisioning/miniprov/hare_mp/templates/hare.init.conf.tmpl.1-node

```diff
@@ -7,9 +7,11 @@
       "storage_set": [
         {
           "durability": {
-            "data": "1",
-            "parity": "0",
-            "spare": "0"
+            "sns": {
```

Comment:
I think for 1-node, 1,0,0 needs to be maintained

Reply:
Agree, since only 2 data devices are there

Reply:
@suvratjoshi Addressed. Please see the updated patch.